frontend/src/component/project/Project/Project.tsx

```diff
@@ -272,20 +272,23 @@ export const Project = () => {
                                         tab.isEnterprise ? 'end' : undefined
                                     }
                                     icon={
-                                        <span>
+                                        <>
                                             <ConditionallyRender
                                                 condition={Boolean(tab.new)}
                                                 show={
-                                                    <StyledBadge color='success'>
-                                                        Beta
-                                                    </StyledBadge>
+                                                    // extra span to avoid badge getting color override from the overly specific parent component
+                                                    <span>
```

Comment:
moving this span around the styled badge to escape css rule with direct tab span child selector that gets the purple color

Reply:
Would it be sensible to update the styling on the tab span instead of adding the extra span? Feels like this fixes the symptom but not the cause. We might need to do the same thing again later? 💁🏼

Reply:
Would probably also make codescene happier

Reply:
Tab is a material UI component with a very specific rule for its direct span child. I'd need to overrider Material UI internals which I didn't want to do.

Reply:
Oh, right. Seems weird that they'd add that, but yeah, happy with your reasoning there.